infoschema/tables.go

```diff
@@ -107,7 +107,9 @@ const (
 	// TableInspectionResult is the string constant of inspection result table
 	TableInspectionResult = "INSPECTION_RESULT"
 	// TableMetricSummary is a summary table that contains all metrics.
-	TableMetricSummary = "METRIC_SUMMARY"
+	TableMetricSummary = "METRICS_SUMMARY"
+	// TableMetricDetail is a detail table that contains all metrics detail info. It is the table metric_summary by label.
+	TableMetricDetail = "METRICS_SUMMARY_BY_LABEL"
```

Comment:
Please update the global variable `TableMetricDetail` name and keep consistent with its value.